### PR TITLE
Added Covid19 awareness website

### PR DESCRIPTION
**Pull request by Sulagna-Dutta-Roy**
I added all the files. This project is made using html , css,  javascript. Add this project under hacktoberfest21.
Thank you in advance.

Issue no : #86 

covidoutput

### CODE REVIEW

**Comment by NOiR-07** [comment]
@Sulagna-Dutta-Roy Make a folder in public and put both the html files in the same folder

**Comment by Sulagna-Dutta-Roy** [comment]
I added html files in separate folder. Please check.

File: Public/Covid19 awareness website/covid19_awareness_protect.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+Stay home and self-isolate even if you have minor symptoms such as cough, headache, mild fever, until you recover. Call your health care provider or hotline for advice. Have someone bring you supplies. If you need to leave your house or have someone near you, wear a medical mask to avoid infecting others.
+If you have a fever, cough and difficulty breathing, seek medical attention immediately. Call by telephone first, if you can and follow the directions of your local health authority.
+Keep up to date on the latest information from trusted sources, such as WHO or your local and national health authorities. Local and national authorities and public health units are best placed to advise on what people in your area should be doing to protect themselves.
+                </li>
+            </ul>
+        </div>
+    </section>
+    <section class="protect" id="protect">
+        <div class="box1">
+            <img src="Images/awarness1.png">
```
Comment (NOiR-07) [inline Public/Covid19 awareness website/covid19_awareness_protect.html:56]: Change all the images src to assets/Images/...

**Comment by NOiR-07** [comment]
@Sulagna-Dutta-Roy Make the changes a i suggested and also you css and js files are not working, change their directory too
for e.g assets/css/foldername

**Comment by NOiR-07** [comment]
@Sulagna-Dutta-Roy Your css isn't working, recheck the directory of folders you mentioned

**Comment by Sulagna-Dutta-Roy** [comment]
Done the changes.

**Pull request closed without merging**